src/app/public/modules/modal/modal.service.ts

```diff
@@ -54,16 +69,8 @@ export class SkyModalService {
     return modalInstance;
   }
 
-  public dispose() {
-    if (SkyModalService.host) {
-      this.dynamicComponentService.removeComponent(SkyModalService.host);
-      SkyModalService.host = undefined;
-    }
-
-  }
-
-  private getConfigFromParameter(providersOrConfig: any) {
-    let defaultParams: IConfig = {
+  private getConfigFromParameter(providersOrConfig: any): any {
```

Comment:
Should this return a known type, instead of `any`?

Reply:
Good call. Done.